### PR TITLE
Branch refactor model

### LINKED ISSUE #21: Update Model for our application
Replace AddressBook class and its respective classes to CommonCents and new/relevant classes for our application.

### PR DESCRIPTION
**Pull request by Ziyang-98**
**Things to take note:**
1. After updating the model and refactoring the model manager, errors pop up in other parts of the code base (in LogicManager, MainApp, commands etc) so I added stubs to those code to avoid compile errors. (Treat it as everything else does not work yet) 

2. I left the previous components of the Model available for reference (AddressBook.java, person package etc) as removing it would result in a huge chunk of errors in other classes since other classes (Ui, Logic, Storage etc) are still using the previous model. The other classes can be refactored to use the current model in future updates.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -16,8 +13,10 @@
 
     @Override
     public CommandResult execute(Model model) {
+        /*
         requireNonNull(model);
-        model.setAddressBook(new AddressBook());
+        model.setCommonCents(new AddressBook());
```
Comment (lettuceman4) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java:18]: Remember to change to new CommonCents() later :") doesnt rlly matter now HAHAH

File: src/main/java/seedu/address/model/account/Account.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                && (name.equals(((Account) other).name)
+                && expenses.equals(((Account) other).expenses)
+                && profits.equals(((Account) other).profits)));
+    }
+
+    @Override
+    public int hashCode() {
+        // use this method for custom fields hashing instead of implementing your own
+        return Objects.hash(name, expenses, profits);
+    }
```
Comment (lettuceman4) [inline src/main/java/seedu/address/model/account/Account.java:194]: I think should include an empty line here (the Addressook they have)

File: src/main/java/seedu/address/model/account/Name.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        return other == this // short circuit if same object
+                || (other instanceof Name // instanceof handles nulls
+                && accountName.equals(((Name) other).accountName)); // state check
+    }
+
+    @Override
+    public int hashCode() {
+        return accountName.hashCode();
+    }
+}
```
Comment (lettuceman4) [inline src/main/java/seedu/address/model/account/Name.java:46]: Uh is there a new line here ah?

**Review by lettuceman4 — COMMENTED** [review]
Got some other new line stuff I only commented once :") but I think roughly ok, can remove the redundant classes (Person, Email, Phone) I think

File: src/main/java/seedu/address/model/account/Account.java (hunk 2 above)
Comment (Ziyang-98) [reply src/main/java/seedu/address/model/account/Account.java:194]: Okay!

File: src/main/java/seedu/address/model/account/Name.java (hunk 3 above)
Comment (Ziyang-98) [reply src/main/java/seedu/address/model/account/Name.java:46]: Okay!

**Review by lettuceman4 — APPROVED** [review]

**Pull request merged**